lang/py/src/avro/schema.py

```diff
@@ -722,7 +748,14 @@ def make_avsc_object(json_data, names=None):
   if hasattr(json_data, 'get') and callable(json_data.get):
     type = json_data.get('type')
     other_props = get_other_props(json_data, SCHEMA_RESERVED_PROPS)
+    logical_type = None
+    if 'logicalType' in json_data:
+      logical_type = json_data.get('logicalType')
+      if logical_type not in ['date']:
+        raise SchemaParseException("Currently does not support %s logical type" % logical_type)
```

Comment:
I wonder if we should do this validation, according to the spec: "Language implementations must ignore unknown logical types when reading, and should use the underlying Avro type. If a logical type is invalid, for example a decimal with scale greater than its precision, then implementations should ignore the logical type and use the underlying Avro type." 

@justinmcunn @premsantosh @abrarsheikh : what do you think?